src/components/OSCALCatalogGroup.js

```diff
@@ -36,51 +38,111 @@ const StyledControlDescriptionWrapper = styled("div")`
   padding: 1em;
 `;
 
-function CollapseableListItem(props) {
+function CollapsibleListItem(props) {
+  const {
+    urlFragment,
+    control,
+    itemText,
+    children,
+    listItemOpened,
+    setListItemOpened,
+  } = props;
   const [open, setOpen] = React.useState(false);
+  const [itemNavigatedTo, setItemNavigatedTo] = React.useState(false);
```

Comment:
For booleans, we should use the `is` convention.

```suggestion
  const [isItemNavigatedTo, setIsItemNavigatedTo] = React.useState(false);
```

We already have a few warnings around this because of `withdrawn`.

Reply:
Gotcha, didn't realize this was the convention of writing booleans. Will keep this in mind.